lib/utils/staticVariables.dart

```diff
@@ -9,23 +9,25 @@ class StaticVariables {
 
   static const String RESSOURCE_PATH = 'res/';
   static const String IMAGE_PATH = RESSOURCE_PATH + 'images/';
+  static const String FAVORITES_REGISTRATION_BUTTON = "Register These Courses";
+  static const String FAVORITES_REGISTRATION_BUTTON = "Register Favorited Courses";
```

Comment:
Both consts have the same name, this will normally not compile.

Reply:
Oops. Sloppy merge on my part :D. Thanks